code/__DEFINES/atmospherics.dm

```diff
@@ -245,6 +245,12 @@
 //DO NOT USE THESE FOR ACCESSING ATMOS DATA, THEY MUTATE THINGS WHEN CALLED. I WILL BEAT YOU WITH A STICK. See the actual proc for more details
 #define CANATMOSPASS(A, O) ( A.CanAtmosPass == ATMOS_PASS_PROC ? A.CanAtmosPass(O) : ( A.CanAtmosPass == ATMOS_PASS_DENSITY ? !A.density : A.CanAtmosPass ) )
 #define CANVERTICALATMOSPASS(A, O) ( A.CanAtmosPassVertical == ATMOS_PASS_PROC ? A.CanAtmosPass(O, TRUE) : ( A.CanAtmosPassVertical == ATMOS_PASS_DENSITY ? !A.density : A.CanAtmosPassVertical ) )
+#define CLEAR_FROM_ATMOS_ADJACENT_TURFS(T) \
+	if (atmos_adjacent_turfs){\
+		atmos_adjacent_turfs -= T;}\
+	if (T.atmos_adjacent_turfs){\
+		T.atmos_adjacent_turfs -= src;}\
+	UNSETEMPTY(T.atmos_adjacent_turfs);
```

Comment:
Might want to put this inside a `do-while(FALSE)` for safety.

Reply:
Cause of UNSETEMPTY?

Reply:
to make sure it doesn't mess with indentation, or has odd interactions with the code you put it in
If you enclose it in a nice {} bundle it's less likely to

Reply:
It's not relevant here, is it a pattern worth establishing you think?